addons/web_editor/models/ir_qweb.py

```diff
@@ -64,18 +63,18 @@ def _compile_node(self, el, options):
                 # The first node might be a call to a sub template
                 sub_call = el.get('t-call')
                 if sub_call:
-                    el.set('t-call-options', f"{{'snippet-key': '{snippet_key}', 'snippet-sub-call-key': '{sub_call}'}}")
+                    el.set('t-options', f"{{'snippet-key': '{snippet_key}', 'snippet-sub-call-key': '{sub_call}'}}")
```

Comment:
IIUC, `t-call-options` is still supported, just using the simpler alias here for consistency